src/app/communities/components/ImageDebris.tsx

```diff
@@ -16,11 +16,12 @@ export const ImageDebris = ({ image }: Props) => {
   const { extractPixelsByConfig } = usePixelExtractor()
   useEffect(() => {
     extractPixelsByConfig(image).then(data => setDebris(data))
-  }, [])
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
   return (
     <>
       {debris.map(debri => (
+        // eslint-disable-next-line @next/next/no-img-element
```

Comment:
Why don't we use the `Image` component?

Reply:
I did it and it worked properly but I did not like the fact that these are random images and we dont know their width and height in advance so we cannot optimize it, therefore there is no benefit of using it.